tempesta_fw/http_msg.c

```diff
@@ -322,8 +266,8 @@ tfw_http_msg_hdr_close(TfwHttpMsg *hm, int id)
 	*h = hm->parser.hdr;
 
 	TFW_STR_INIT(&hm->parser.hdr);
-	TFW_DBG3("store header w/ ptr=%p len=%lu flags=%x id=%d\n",
-		 h->ptr, h->len, h->flags, id);
+	TFW_DBG3("store header w/ ptr=%p len=%lu eolen=%u flags=%x id=%d\n",
+		 h->ptr, h->len, h->eolen, h->flags, id);
 
 	/* Move the offset forward if current header is fully read. */
 	if (id == ht->off)
```

Comment:
Yes, the issues by results of static analyzers are appreciated. We use [Coverity](https://scan.coverity.com/projects/tempesta-tech-tempesta), but we didn't upload our source code for a while, so making the new verification has sense.

Basically we don't use Clang for kernel source code, but we still can use its static analyzing. I leave the code as is for now since I have no Clang and never used it before. However, it would be very useful if you upload the new code to Coverity and create a task for all Clang warnings.